frontend/src/Page2.js

```diff
@@ -0,0 +1,114 @@
+/** @jsx jsx */
+import { navigate } from '@reach/router'
+import { Trans } from '@lingui/macro'
+import { Form, Field } from 'react-final-form'
+import { H1 } from './components/header'
+import { Ul } from './components/unordered-list'
+import { Li } from './components/list-item'
+import { ApolloConsumer } from 'react-apollo'
+import { TrackPageViews } from './TrackPageViews'
+import { i18nMark } from '@lingui/react'
+import { Container } from './components/container'
+import { jsx, css } from '@emotion/core'
+import { TextArea } from './components/text-area'
+import { Button } from './components/button'
+import { Label } from './components/label'
+import { P, Lead } from './components/paragraph'
+
+/* eslint-disable-next-line */
+const submitAndNavigate = (client, data) => {
+  // client.writeData({ data })
+  // console.log('hit submit')
```

Comment:
🔥 (at least the console.log)

Reply:
just taking out the log because we'll want the other stuff later